### PR TITLE
Zoom.Zoom version 5.13.12602

### PR DESCRIPTION
**Pull request by mivsvit**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/95962)

### CODE REVIEW

File: manifests/z/Zoom/Zoom/5.17.12602/Zoom.Zoom.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,97 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Zoom.Zoom
+PackageVersion: 5.17.12602
```
Comment (Trenly) [inline manifests/z/Zoom/Zoom/5.17.12602/Zoom.Zoom.installer.yaml:5]:
```suggestion
PackageVersion: 5.13.12602
```

File: manifests/z/Zoom/Zoom/5.17.12602/Zoom.Zoom.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Zoom.Zoom
+PackageVersion: 5.17.12602
```
Comment (Trenly) [inline manifests/z/Zoom/Zoom/5.17.12602/Zoom.Zoom.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 5.13.12602
```

File: manifests/z/Zoom/Zoom/5.17.12602/Zoom.Zoom.locale.zh-CN.yaml [hunk 3]
```diff
@@ -0,0 +1,30 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.4.0.schema.json
+
+PackageIdentifier: Zoom.Zoom
+PackageVersion: 5.17.12602
```
Comment (Trenly) [inline manifests/z/Zoom/Zoom/5.17.12602/Zoom.Zoom.locale.zh-CN.yaml:5]:
```suggestion
PackageVersion: 5.13.12602
```

File: manifests/z/Zoom/Zoom/5.17.12602/Zoom.Zoom.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: Zoom.Zoom
+PackageVersion: 5.17.12602
```
Comment (Trenly) [inline manifests/z/Zoom/Zoom/5.17.12602/Zoom.Zoom.yaml:5]:
```suggestion
PackageVersion: 5.13.12602
```

**Review by Trenly — CHANGES_REQUESTED** [review]
```
--> Comparing ARP Entries

DisplayName DisplayVersion Publisher ProductCode
----------- -------------- --------- -----------
Zoom(64bit) 5.13.12602     Zoom      {099FDE51-11D7-43C4-9AAB-635C8F898826}
```

**Comment by stephengillie** [comment]
Hi @misvit,

The path has a "17" where the version has a "13". Validation is erroring because they do not match:

>  Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. 
> actual 'manifests\z\Zoom\Zoom\5.17.12602', 
> expected 'manifests\z\Zoom\Zoom\5.13.12602'

Can you modify the manifest file paths in your PR to match the version?

**Pull request closed without merging**